code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -257,6 +257,7 @@
 	AddElement(/datum/element/atmos_sensitive, mapload)
 	become_hearing_sensitive(trait_source = ROUNDSTART_TRAIT)
 	ADD_TRAIT(src, TRAIT_ASHSTORM_IMMUNE, ROUNDSTART_TRAIT) //protects pilots from ashstorms.
+	ADD_TRAIT(src, TRAIT_SNOWSTORM_IMMUNE, ROUNDSTART_TRAIT) //protects pilots from snowstorms
```

Comment:
should use `add_traits()` proc here instead

Reply:
https://github.com/tgstation/tgstation/pull/76367/commits/a65c54380b79856161f042425225bd1d54627e4c